src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -0,0 +1,128 @@
+
+    /**
+     * Upgrade a builder to lv 1
+     */
+    public static final Achievement achievementBuildingBuilder   = new MineColoniesAchievement("upgrade.builder.first", "upgrade.builder.first", 0, 1, ModBlocks.blockHutBuilder, achievementBuildingTownhall).registerStat();
+    /**
+     * Max out a builder
+     */
+    public static final Achievement achievementUpgradeBuilderMax = new MineColoniesAchievement("upgrade.builder.max", "upgrade.builder.max", 2, 1, ModBlocks.blockHutBuilder, achievementBuildingBuilder).registerStat();
+
+    /**
+     * Upgrade a builder to lv 1
+     */
+    public static final Achievement achievementBuildingColonist   = new MineColoniesAchievement("upgrade.colonist.first", "upgrade.colonist.first", 0, 2, ModBlocks.blockHutCitizen, achievementGetSupply).registerStat();
+    /**
+     * Max out a builder
+     */
+    public static final Achievement achievementUpgradeColonistMax = new MineColoniesAchievement("upgrade.colonist.max", "upgrade.colonist.max", 2, 2, ModBlocks.blockHutCitizen, achievementBuildingColonist).registerStat();
+
+    /**
+     * Upgrade a lumberjack to lv 1
+     */
+    public static final Achievement achievementBuildingLumberjack   = new MineColoniesAchievement("upgrade.lumberjack.first", "upgrade.lumberjack.first", 0, 3, ModBlocks.blockHutLumberjack, achievementGetSupply).registerStat();
+    /**
+     * Max out a lumberjack
+     */
+    public static final Achievement achievementUpgradeLumberjackMax = new MineColoniesAchievement("upgrade.lumberjack.max", "upgrade.lumberjack.max", 2, 3, ModBlocks.blockHutLumberjack, achievementBuildingLumberjack).registerStat();
+
+    /**
+     * Upgrade a miner to lv 1
+     */
+    public static final Achievement achievementBuildingMiner   = new MineColoniesAchievement("upgrade.miner.first", "upgrade.miner.first", 0, 4, ModBlocks.blockHutMiner, achievementGetSupply).registerStat();
+    /**
+     * Max out a miner
+     */
+    public static final Achievement achievementUpgradeMinerMax = new MineColoniesAchievement("upgrade.miner.max", "upgrade.miner.max", 2, 4, ModBlocks.blockHutMiner, achievementBuildingMiner).registerStat();
+
+    /**
+     * Upgrade a fisher to lv 1
+     */
+    public static final Achievement achievementBuildingFisher   = new MineColoniesAchievement("upgrade.fisher.first", "upgrade.fisher.first", 0, 5, ModBlocks.blockHutFisherman, achievementGetSupply).registerStat();
+    /**
+     * Max out a fisher
+     */
+    public static final Achievement achievementUpgradeFisherMax = new MineColoniesAchievement("upgrade.fisher.max", "upgrade.fisher.max", 2, 5, ModBlocks.blockHutFisherman, achievementBuildingFisher).registerStat();
+
+    // Sizes
+    /**
+     * Reach {@link AchievementSizeSettlement#SIZE} citizens
+     */
+    public static final Achievement achievementSizeSettlement = new MineColoniesAchievement("size.pioneer", "size.settlement", 2, -2, Items.iron_ingot, null).registerStat();
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)